packages/platforms/browser/tests/on-settle/index.test.ts

```diff
@@ -357,7 +357,7 @@ describe('onSettle', () => {
     expect(settleCallback).not.toHaveBeenCalled()
 
     // request should be ignored, so advancing by 100ms will settle
-    fetchRequestTracker.start({ ...START_CONTEXT, url: 'https://otlp.bugsnag.com/v1/traces' })
+    fetchRequestTracker.start({ ...START_CONTEXT, url: `https://${VALID_API_KEY}.otlp.bugsnag.com/v1/traces` })
```

Comment:
without having dug too far into this test: are we hitting the trace server in a unit test?

Reply:
i don't think so, requestTracker doesn't do any sending and there is no "delivery", real or mock used in this test